packages/@sfx/core/test/unit/common/core.test.ts

```diff
@@ -113,5 +204,14 @@ describe('Core', () => {
         sinon.match.same(directory)
       );
     });
+
+    it('should clear dependency graph', () => {
+      const unregisterPlugins = stub(CoreUtils, 'unregisterPlugins');
```

Comment:
Does this need to be stored in a variable if it's not being used?

Reply:
It does not. I'll remove it.